ZeekPluginCommon.cmake

```diff
@@ -20,6 +27,24 @@ function(zeek_plugin_begin ns name)
     set(_plugin_deps       "" PARENT_SCOPE)
     set(_plugin_dist       "" PARENT_SCOPE)
     set(_plugin_scripts    "" PARENT_SCOPE)
+
+    cmake_parse_arguments(PARSE_ARGV 2 ZEEK_PLUGIN "DISABLE_CPP_TESTS" "" "")
+
+    # Whether to build the plugin with unit-test support
+    set(_plugin_cpp_tests true PARENT_SCOPE)
```

Comment:
Just a nit: want to move this into the `if` body? Confused me here at first to see it set to true unconditionally, until I realized the `else` branch does the `false`. If you move it, the if/else branches become symmetric.